instagram/src/App.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './utilities/dummy_data';
+import PostContainer from './components/PostContainer/PostContainer';
+import SearchBar from './components/SearchBar/SearchBar';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      posts: dummyData,
+      searchQuery: ''
+    }
+  }
+
+  onSearch(searchQuery) {
+    this.setState({
+      ...this.state,
+      searchQuery
+    })
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <SearchBar onSearch={(searchQuery) => this.onSearch(searchQuery)} />
```

Comment:
I'm curious why you're using an arrow function to wrap this function here. It doesn't seem to serve any purpose besides getting around `this`, so why not pass just the function definition and also make that an arrow function?
```javascript
onSearch = searchQuery => {
  this.setState({
    ...this.state,
    searchQuery
  });
};
// ...
<SearchBar onSearch={this.onSearch} />
```